diff --git a/usr/share/lipstick-jolla-home-qt5/compositor.qml b/usr/share/lipstick-jolla-home-qt5/compositor.qml
index 2a3c9cc..2a0abe0 100644
--- a/usr/share/lipstick-jolla-home-qt5/compositor.qml
+++ b/usr/share/lipstick-jolla-home-qt5/compositor.qml
@@ -851,7 +851,7 @@ Compositor {
 
                 sourceItem: used ? transitioningLayer.snapshotSource : null
                 hideSource: true
-                live: false
+                live: true
 
                 visible: used || transitionOptimizerFadeOut.running
                 opacity: 0
